extra_stuff/lessons_in_progress/join_the_odin_community.md

```diff
@@ -0,0 +1,64 @@
+# Join the Odin Community
+
+Working and collaborating with other people on a project, whether it be big or small, is an important part of becoming a web developer. Therefore at Odin we encourage you to participate in our [online community chat](https://gitter.im/orgs/TheOdinProject/rooms) where we can all grow together towards becoming web developers. 
+
+<!-- Call to Action GITTER SIGN IN BUTTON using this link https://gitter.im/orgs/TheOdinProject/rooms -->
+
+## Introductions: 'Hello World!'
+Jump in and introduce yourself; tell us your story! Where are you from? What made you aspire to become a web developer? What are your hopes and dreams? We encourage you to keep on sharing your thoughts AND your code.
+
+### Why chat is awesome for you:
+
+Web development will be a long and arduous journey - you can make the marathon a lot more fun by pairing up with somebody who is running it at the same pace as you. It is a great way to bounce ideas off each other and motivate one another to the finish line.
+
+Veterans at The Odin Project love to help fill in knowledge gaps and provide new insights and perspectives on how to improve your code. Helping others that haven't made it as far as you is a great way to deepen your own understanding and make your learning stick.
+
+When you run into the desert of despair - where your code doesn't work or make sense to you anymore - you can find an oasis of knowledge and encouragement in our chat. We've all been there before!
+
+Remember that project you struggled with tirelessly and that you're so proud of finishing? You will get to share your work and progress with those that really appreciate how much hard work went into it.
+
+### Why chat is awesome for Odin:
+
+We are working hard to update existing lessons and produce new content so we would love to hear your feedback on how you are coping with the lessons and projects. We really want you to find the lessons fun, engaging and informative; while we would like you to find projects to be doable and challenging. So please tell us!
+
+## Oh foo bar!
+
+With great projects come great responsibility. When embarking on your projects, keep in mind that you do not need to know everything - BUT you do need to understand where you are going! Try to be as purposeful and specific as possible when describing the problem. Sometimes the problem can be a matter of your approach and not your coding ability: keep an open mind at all times.
```

Comment:
I see what you mean about this paragraph. I think it will be a bit ambiguous and confusing for a beginner. I don't really have any good ideas for making it clearer at the moment. @codyloyd, @cdouglass @kriox26 we would love your input.